ts/Core/Legend/LegendOptions.d.ts

```diff
@@ -75,6 +75,7 @@ export interface LegendOptions {
     /** @deprecated */
     lineHeight?: number;
     margin?: number;
+    maxWidth?: number;
```

Comment:
This one accepts a percentage string too, right?
```suggestion
    maxWidth?: number|string;
```